docs/api-reference.md

```diff
@@ -94,6 +94,40 @@ curl "localhost:10101/index/user/query?columnAttrs=true&shards=0,1" \
 
 By default, all bits and attributes (*for `Row` queries only*) are returned. In order to suppress returning bits, set `excludeBits` query argument to `true`; to suppress returning attributes, set `excludeAttrs` query argument to `true`.
 
+### Import Data
+
+`POST /index/<index-name>/field/<field-name>`
+
+Supports high-rate data ingest to a particular shard of a particular field. The
+official client libraries use this endpoint for their import functionality - it
+is not usually necessary to use this endpoint directly. See the documentation for
+imports for
+<a href="https://github.com/pilosa/go-pilosa/blob/master/docs/imports-exports.md">Go</a>,
+<a href="https://github.com/pilosa/java-pilosa/blob/master/docs/imports.md">Java</a>,
+and <a href="https://github.com/pilosa/python-pilosa/tree/master/docs/imports.md">Python</a>.
+
+The request payload is protobuf encoded with the following schema. The row or
+column Keys fields are used if the field or index is configured for keys
```

Comment:
This means "the fields `RowKeys` or `ColumnKeys` ...", right? The explicit phrasing is a bit clearer to me.